### PR TITLE
Fix browser detection for Edge

### LINKED ISSUE #71: Edge detection is broken for legacy Edge browser
We should be showing the unsupported browser version of the page to visitors using legacy Edge.

Instead we show the regular "install for Edge" version.

### PR DESCRIPTION
**Pull request by wioux**
Fixes #71.

### CODE REVIEW

File: assets/js/download.js [hunk 1]
```diff
@@ -15,7 +16,7 @@ $(document).ready(function() {
     else if (navigator.userAgent.match(/android/i)) {
       browser = 'other';
     }
-    else if (navigator.userAgent.match(/chrome\/.+edge?\//i)) {
+    else if ((m = navigator.userAgent.match(/chrome\/.+edge?\/(\d+)/i)) && parseInt(m[1]) >= 79) {
```
Comment (ghostwords) [inline assets/js/download.js:19]: Do we need to check version numbers? It looks like ChrEdge uses "Edg/" while legacy Edge uses "Edge/".

File: assets/js/download.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ $(document).ready(function() {
     else if (navigator.userAgent.match(/android/i)) {
       browser = 'other';
     }
-    else if (navigator.userAgent.match(/chrome\/.+edge?\//i)) {
+    else if (navigator.userAgent.match(/chrome\/.+edg\//)) {
```
Comment (ghostwords) [inline assets/js/download.js:18]: Why no longer case insensitive?

Comment (wioux) [reply assets/js/download.js]: Sorry. Thanks for catching that.

File: assets/js/download.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ $(document).ready(function() {
     else if (navigator.userAgent.match(/android/i)) {
       browser = 'other';
     }
-    else if (navigator.userAgent.match(/chrome\/.+edg\//)) {
+    else if (navigator.userAgent.match(/chrome\/.+edg\/i/)) {
```
Comment (ghostwords) [inline assets/js/download.js:18]: The "i" qualifier is in the wrong place.

**Comment by ghostwords** [comment]
@wioux Let me know when this is ready for review.

**Pull request merged**